server/configmodule.py

```diff
@@ -57,7 +58,8 @@ class PrivateConfig(Config):
 
 class FeedingAmericaConfig(PrivateConfig):
     NAME = "Feeding America"
-    FEEDING_AMERICA = True
+    ENV_NAME = 'FEEDING_AMERICA'
+    BASE_HTML_PATH = 'private_dc/feeding_america/base.html'
```

Comment:
This is nice! Note after merge, should change to "feedingamerica" throughout

Reply:
With this change, I think we can remove "PrivateConfig" class above which sets "PRIVATE=True". There PRIVATE is really an environment name. Note here FeedingamericaConfig inherits that. Then this is used in static.py when checking the env

Reply:
+1 - updated